Client.Test.Integration/WriteTest.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Frozen;
+using System.Linq;
+using System.Threading.Tasks;
+using InfluxDB3.Client.Config;
+using NUnit.Framework;
+
+namespace InfluxDB3.Client.Test.Integration;
+
+public class WriteTest : IntegrationTest
+{
+
+    [Test]
+    public async Task WriteWithError()
+    {
+        using var client = new InfluxDBClient(new ClientConfig
+        {
+            Host = Host,
+            Token = Token,
+            Database = Database,
+        });
+
+        try
+        {
+            await client.WriteRecordAsync("vehicle,id=vwbus vel=0.0,distance=,status=\"STOPPED\"");
+        }
+        catch (Exception ex)
+        {
+            if (ex is InfluxDBApiException)
+            {
+                var iaex = (InfluxDBApiException)ex;
+                Assert.Multiple(() =>
+                {
+                    Assert.That(iaex.Message,
+                        Is.EqualTo(
+                            "errors encountered on line(s): line 1: " +
+                            "Could not parse entire line. Found trailing content: 'distance=,status=\"STOPPED\"'"
+                        ));
+                    Assert.That(iaex.GetStatusCode().ToString(), Is.EqualTo("BadRequest"));
+                    Assert.That(iaex.GetStatusCode().GetHashCode(), Is.EqualTo(400));
```

Comment:
There is System.Net enum for status code, eg.

```suggestion
                    Assert.That(iaex.GetStatusCode(), Is.EqualTo(System.Net.HttpStatusCode.BadRequest));
```

Reply:
Updated, following suggestion.